### PR TITLE
Onboarding final final fixes

### PR DESCRIPTION
**Pull request by Gnito**
Juho wanted to change his earlier copys + found couple of bugs

### CODE REVIEW

File: app/controllers/admin/getting_started_guide_controller.rb [hunk 1]
```diff
@@ -19,6 +19,9 @@ def data
     has_sub_path = (path_parts.count == 2 && path_parts[1] != "/")
     sub_path = has_sub_path ? path_parts[1] : ""
 
+    listing_shape = ListingShape.where(community_id: @current_community.id).first
```
Comment (kpuputti) [inline app/controllers/admin/getting_started_guide_controller.rb]: The other place that used the same expression `ListingShape.where(...).first` wrapped it within a Maybe, should this do that as well? Does this assume a shape is always found for a community? I don't know if anything else is possible, that's why I'm asking.

File: app/controllers/admin/getting_started_guide_controller.rb [hunk 2]
```diff
@@ -19,6 +19,10 @@ def data
     has_sub_path = (path_parts.count == 2 && path_parts[1] != "/")
     sub_path = has_sub_path ? path_parts[1] : ""
 
+    alternative_cta = Maybe(ListingShape.where(community_id: @current_community.id).first)
```
Comment (rap1ds) [inline app/controllers/admin/getting_started_guide_controller.rb]: `ListingShape` model shouldn't be referenced directly from a controller. You can use the `ListingService::API::Api.shapes.get(community_id: @current_community.id)` method.

File: app/controllers/styleguide/pages_controller.rb [hunk 3]
```diff
@@ -29,6 +29,10 @@ def data
       invitation: true
     }
 
+    alternative_cta = Maybe(ListingShape.where(community_id: @current_community.id).first)
```
Comment (rap1ds) [inline app/controllers/styleguide/pages_controller.rb]: Same here.

File: app/controllers/admin/getting_started_guide_controller.rb (hunk 2 above)
Comment (rap1ds) [inline app/controllers/admin/getting_started_guide_controller.rb]: Btw. how do you know the `first` order type is the one that needs to be edited?

Comment (Gnito) [inline app/controllers/admin/getting_started_guide_controller.rb]: Ah, true.
(btw. part: that's how Juho wanted this to work.)

File: config/locales/onboarding/en.yml [hunk 4]
```diff
@@ -53,7 +53,7 @@ en:
           title_done: "All done!"
           description_p1: "To get your marketplace up and running, there are a few essential steps you need to take, as listed below."
           description_p2: "Once finished, your marketplace will be ready to receive its first visitors!"
-          congratulation_p1: "You did it! Your marketplace is now up and running. However, this is just the beginning. You've only seen a fraction of the customization options that Sharetribe offers. To read more about how to customize your marketplace further, check out <a href='http://support.sharetribe.com/knowledgebase' alt='knowledge base' target='_blank'>our knowledge base</a>."
+          congratulation_p1: "You did it! Your marketplace is now up and running. However, this is just the beginning. You've only seen a fraction of the customization options that Sharetribe offers. To read more about how to customize your marketplace further, check out <a href='http://support.sharetribe.com/knowledgebase/articles/892140-what-to-do-after-the-basic-setup-of-your-marketpla' alt='knowledge base' target='_blank'>our knowledge base</a>."
```
Comment (rap1ds) [inline config/locales/onboarding/en.yml]: HTML in the translations :(

**Pull request merged**